### DISCUSSION TITLE
[bug report] Changing filters does not change pagination

### DISCUSSION THREAD

**Discussion by aviggiano** [solodit/feedback#10] [New Feature Requests]
Steps to reproduce:

1. Filter by keyword (e.g. liquidation)
2. Go to page N (e.g. page 5)
3. Change filter (e.g. set keyword to lending)
4. Page will still be N, while it should have been changed to 1

**Comment by hans-cyfrin** [comment]
@aviggiano We could not reproduce this bug.
Please note that after changing filter you would need to hit the Search again to get the updated filter results.
Please let us know if you still the problem.

**Reply by aviggiano** [reply]
Well actually I couldn't reproduce it either. 
I think it was me not clicking clear filters, maybe. I'll close this issue
